addons/mail/models/mail_thread.py

```diff
@@ -121,15 +121,18 @@ def _compute_message_partner_ids(self):
     def _search_message_partner_ids(self, operator, operand):
```

Comment:
Commit message line is too long, should be ~75/80 characters-long

`, therefor, ` -> `. Therefore,`